packages/swarm-components/src/shared/CharCount.jsx

```diff
@@ -0,0 +1,12 @@
+// @flow
+import * as React from 'react';
+
+type CharProps = {
+    children?: React.Node
+}
+
+export const getRemainingCharacters = (maxLength: number, value: string = '') => (parseInt(maxLength, 10) - value.length);
```

Comment:
currently, `maxLength` can also be a `string` - I think Flow should be complaining, so we might need to find out why it isn't...

Reply:
hunter filed a ticket to look into this. there's actually a bunch of flow errors on master as well. i'll update the ticket to fix the flow errors.